main.tf

```diff
@@ -21,6 +21,7 @@ resource "ibm_iam_authorization_policy" "policy" {
   target_service_name         = "secrets-manager"
   target_resource_instance_id = var.secrets_manager_id
   roles                       = ["SecretsReader"]
+  description                 = "Allow all server route instances in the resource group ${var.resource_group_id} to read secrets-manager instance GUID ${var.secrets_manager_id}"
```

Comment:
Allow all VPN server instances in the resource group ${var.resource_group_id} to read from the Secrets Manager instance with ID ${var.secrets_manager_id}